app/utils/highcharts.js

```diff
@@ -0,0 +1,31 @@
+import { assign } from '@ember/polyfills';
+import merge from 'deepmerge';
+
+function createChartOptions({
+  chartOptions = {},
+  data = [{ color: '#aaaaaa', data: 0, id: 'placeholder' }],
+  theme = {},
+} = {}) {
+  const themePlusChartOptions = merge(theme, chartOptions);
+
+  return assign({ series: data }, themePlusChartOptions);
+}
+
+const highchartsOptions = {
+  credits: {
+    enabled: false,
+  },
+
+  plotOptions: {
+    series: {
+      animation: false,
```

Comment:
I disabled animation to make writing tests easier.

Allowing animation on _all_ charts also felt unnecessary when the user may see no charts or only 1 chart on the top of their screen when the page loads.